nbuild/00.Foreword.md

```diff
@@ -0,0 +1,16 @@
+# Foreword
+Welcome to Nest build's official getting-started guide!
+
+Nest build, or `nbuild`, is an automated package builder for [Raven-OS](https://raven-os.org), which is an adaptative, customizable and evolutive open-source Operating System (OS) based on the Linux kernel. You can use `nbuild` on any linux-based distribution though.
```

Comment:
I don't like the use of "though" at the end of the sentence, but can't seem to find a good replacement. The sentence coule be less personnal too ("*`nbuild` can be run [...]*").